packages/atomic/src/components/atomic-breadbox/atomic-breadbox.tsx

```diff
@@ -0,0 +1,247 @@
+          class="p-2 btn-pill"
+          onClick={() => (this.isCollapsed = true)}
+        ></Button>
+      </li>
+    );
+  }
+
+  private renderClearAll() {
+    return (
+      <li>
+        <Button
+          part="clear"
+          style="text-primary"
+          text={this.bindings.i18n.t('clear')}
+          class="p-2 btn-pill"
+          ariaLabel={this.bindings.i18n.t('clear-all-filters')}
+          onClick={() => this.breadcrumbManager.deselectAll()}
+        ></Button>
+      </li>
+    );
+  }
+
+  private get allBreadcrumbs(): Breadcrumb[] {
+    return [
+      ...this.breadcrumbManagerState.facetBreadcrumbs
+        .map(({facetId, field, values}) =>
+          values.map((value) => ({value, facetId, field}))
+        )
+        .flat()
+        .map(({value, facetId, field}) => ({
+          facetId,
+          label: this.bindings.store.state.facets[facetId].label,
+          deselect: value.deselect,
+          formattedValue: getFieldValueCaption(
+            field,
+            value.value.value,
+            this.bindings.i18n
+          ),
+        })),
+      ...this.breadcrumbManagerState.categoryFacetBreadcrumbs.map(
+        ({facetId, field, path, deselect}) => ({
+          facetId,
+          label: this.bindings.store.state.categoryFacets[facetId].label,
+          deselect: deselect,
+          formattedValue: path.map((pathValue) =>
+            getFieldValueCaption(field, pathValue.value, this.bindings.i18n)
+          ),
+        })
+      ),
+      ...this.breadcrumbManagerState.numericFacetBreadcrumbs
+        .map(({facetId, field, values}) =>
+          values.map((value) => ({value, facetId, field}))
+        )
+        .flat()
+        .map(({value, facetId}) => ({
+          facetId,
+          label: this.bindings.store.state.numericFacets[facetId].label,
+          deselect: value.deselect,
+          formattedValue: this.bindings.store.state.numericFacets[
+            facetId
+          ].format(value.value),
+        })),
+      ...this.breadcrumbManagerState.dateFacetBreadcrumbs
+        .map(({facetId, field, values}) =>
+          values.map((value) => ({value, facetId, field}))
+        )
+        .flat()
+        .map(({value, facetId}) => ({
+          facetId,
+          label: this.bindings.store.state.dateFacets[facetId].label,
+          deselect: value.deselect,
+          formattedValue: this.bindings.store.state.dateFacets[facetId].format(
+            value.value
+          ),
+        })),
+    ];
+  }
+
+  private renderBreadcrumbs() {
+    const sortedBreadcrumbs = this.allBreadcrumbs.sort((a, b) => {
+      const indexA = this.facetManagerState.facetIds.indexOf(a.facetId);
+      const indexB = this.facetManagerState.facetIds.indexOf(b.facetId);
+      return indexA - indexB;
+    });
+    const slicedBreadcrumbs = this.isCollapsed
+      ? sortedBreadcrumbs.slice(0, this.collapseThreshold)
+      : sortedBreadcrumbs;
+
+    return [
```

Comment:
Similar comment as above, but just old boring code that is not as condensed with small named function would be easier to read.